syntax = "proto3";

service UnusualAction {
  rpc InformUnusualAction(InformUnusualActionRequest) returns (InformUnusualActionResponse);
}

message InformUnusualActionRequest {
  string type = 1;
  string description = 2;
  string occurAt = 3;
  int64 roomId = 4;
  int64 userId = 5;
}

message InformUnusualActionResponse {
  string message = 1;
}